catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1125,19 +1134,44 @@ public DeleteResponse delete(DeleteRequest deleteRequest)
 
         DeleteResponse deleteResponse = null;
         try {
+            List<Filter> idFilters = new ArrayList<>();
+            for (Serializable serializable : deleteRequest.getAttributeValues()) {
```

Comment:
Yeah, I actually had that at one point and then changed it back because it just seemed like it was harder to read to me because of the filter builder stuff in there.